homeassistant/components/nintendo_parental/sensor.py

```diff
@@ -0,0 +1,94 @@
+"""Sensor platform for Nintendo Parental."""
+
+from __future__ import annotations
+
+from collections.abc import Callable, Mapping
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+
+from .coordinator import NintendoParentalConfigEntry, NintendoUpdateCoordinator
+from .entity import Device, NintendoDevice
+
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
+
+
+@dataclass(kw_only=True, frozen=True)
+class NintendoParentalSensorEntityDescription(SensorEntityDescription):
+    """Description for Nintendo Parental sensor entities."""
+
+    value_fn: Callable[[Device], int | float | None]
+    state_attributes_fn: Callable[[Device], Mapping[str, Any] | None]
+
+
+SENSOR_DESCRIPTIONS: tuple[NintendoParentalSensorEntityDescription, ...] = (
+    NintendoParentalSensorEntityDescription(
+        key="playing_time",
+        name="used screen time",
```

Comment:
don't hardcode the entity name. I think it did not work as expected because the `translation_key` was missing
```suggestion
        name="used screen time",
```